src/components/Sidebar/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faIdCard } from '@fortawesome/free-solid-svg-icons';
+import { faClock } from '@fortawesome/free-solid-svg-icons';
+import { faUsers } from '@fortawesome/free-solid-svg-icons';
+import { faProjectDiagram } from '@fortawesome/free-solid-svg-icons';
+import { faChartLine } from '@fortawesome/free-solid-svg-icons';
+import { faStreetView } from '@fortawesome/free-solid-svg-icons';
+import styles from './Sidebar.module.css';
+
+const Sidebar = () => {
```

Comment:
The Sidebar will look different from admin's point of view, so the sidebar items should be generic
To achieve this, please extract sidebar item data to an array, for example
```
const sideBarItems = [
     {
           icon: faStreetView,
           className: styles.icon,
           label: 'Overview',
       },
   ]